ansible/roles/nginx/templates/nginx.conf.j2

```diff
@@ -37,6 +37,7 @@ http {
         }
         location /api/v1 {
             proxy_pass http://{{ groups['controllers']|first }}:{{ controller.port }};
+            proxy_read_timeout 300s;
```

Comment:
So it's going to be 305 (including grace) + some nanoseconds at least to get the request back to nginx so it doesn't say: Gateway Timeout?